shared/src/business/useCases/serveCaseToIrs/serveCaseToIrsInteractor.test.js

```diff
@@ -162,9 +142,7 @@ describe('serveCaseToIrsInteractor', () => {
         userId: '6805d1ab-18d0-43ec-bafb-654e83405416',
       }),
     );
-    applicationContext
-      .getPersistenceGateway()
-      .getCaseByDocketNumber.mockReturnValue(MOCK_CASE);
+    mockCase = MOCK_CASE;
```

Comment:
Hmm, this seems possibly wonky. If any of these tests mutate mockCase, MOCK_CASE would also get changed. Wouldn't we need to spread MOCK_CASE?

```
mockCase = { ...MOCK_CASE };
```

Reply:
Good catch!

Reply:
Updated!